docs/about/release-notes.md

```diff
@@ -1,5 +1,35 @@
 # Release Notes
 
+
+## ShapeWorks 6.2.1 - 2022-01-07
+
+### What is new?
+
+#### Studio
```

Comment:
For the studio list, I'd rather convert this list to one or more bulleted major features added to Studio (please see previous notes). For example, *New mesh grooming tool in Studio" then add the relevant list items him to this one single major point.